clamav-bytecode-compiler/1.4/ubuntu/Dockerfile

```diff
@@ -19,15 +19,13 @@ RUN apt-get update -y && \
         binutils \
         git \
         python3 \
-        python3-distutils \
-        python3-pip \
```

Comment:
I don't understand why, but removing the python3-pip gives me warnings that pytest is not installed.